azerbaijani/nağıl

```diff
@@ -0,0 +1,7 @@
+Biri var idi, biri yox idi. Bir ayı, günü keçmiş kişi var idi. Bir arvadı var idi. Bunların yeddi oğlanları var idi. Amma heç qız uşaqları yox idi. Yeddi qardaşlar bir bacıları olmadığından çox qəmgin idilər. Bir gün yeddi qardaşlar analarına dedilər:
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.